serverless-functions/change/serverless.yml

```diff
@@ -0,0 +1,39 @@
+service: change-service
```

Comment:
sometimes i feel like we need to template the template hehe. maybe we'll end up with a readme of where all the vars are?